src/accelerate/state.py

```diff
@@ -238,9 +241,13 @@ def __init__(self, cpu: bool = False, **kwargs):
                     torch.distributed.init_process_group(self.backend, rank=rank, world_size=size, **kwargs)
                 self.num_processes = torch.distributed.get_world_size()
                 self.process_index = torch.distributed.get_rank()
-                self.local_process_index = local_rank
```

Comment:
This shouldn't be removed, no?

Reply:
> This shouldn't be removed, no?

thanks for review, I forgot to mention that I removed it is because it is deprecated in line 225, https://github.com/huggingface/accelerate/pull/1508/files#diff-e54d1f9cdd342420827fcabd4280eceb68da266c9ca6df120d0432c6f6f2e10aR225